scripts/ci_steps/codegen_sdk.sh

```diff
@@ -12,6 +12,7 @@ rm -rf Tests/Services/*
 ./gradlew --stop
 
 # Regenerate the SDK Package.swift with all services
+unset AWS_SWIFT_SDK_USE_LOCAL_DEPS
```

Comment:
`AWS_SWIFT_SDK_USE_LOCAL_DEPS` is unset here & below before running the CLI tools so that the SDK used by the build tools doesn't attempt to use local `smithy-swift`.